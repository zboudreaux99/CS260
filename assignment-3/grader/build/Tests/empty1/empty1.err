Memory leak detected: 600
