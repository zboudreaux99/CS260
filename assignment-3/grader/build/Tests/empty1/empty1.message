Output line 0:
  <i>expected: </i>emptyList.txt contained 0 books. mediumList2.txt contained <span class='expected'>1000</span> books. Their merge contains 1000 books. etext10026 Various The 
  <i>observed: </i>emptyList.txt contained 0 books. mediumList2.txt contained <span class='observed'>0</span> books. Their merge contains 0 books. --- end of output --- 
