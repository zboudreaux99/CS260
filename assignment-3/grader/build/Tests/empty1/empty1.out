emptyList.txt contained 0 books.
mediumList2.txt contained 0 books.
Their merge contains 0 books.
